core/src/control/precice/nested_solver_fast_monodomain_solver.tpp

```diff
@@ -581,7 +581,11 @@ void PreciceAdapterNestedSolver<FastMonodomainSolver<T1>>::getTractionValues(
 
 template <typename T1>
 Vec PreciceAdapterNestedSolver<FastMonodomainSolver<T1>>::currentState(
-    NestedSolverType &nestedSolver) {}
+    NestedSolverType &nestedSolver) {
+  LOG(ERROR) << "Implicit coupling not supported! curentState is not "
```

Comment:
Will this also trigger a controlled exit?

Reply:
Yes, I checked:)